add-graphics-renderer/src/main/java/com/esri/arcgisruntime/sample/addgraphicsrenderer/MainActivity.java

```diff
@@ -46,65 +45,71 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // inflate MapView from layout
         mMapView = (MapView) findViewById(R.id.mapView);
-
         // create a map with the Basemap Type topographic
-        Map mMap = new Map(Basemap.Type.TOPOGRAPHIC, 34.056295, -117.195800, 14);
+        Map mMap = new Map(Basemap.Type.TOPOGRAPHIC, 15.169193, 16.333479, 2);
+        // add graphics overlay
+        addGraphicsOverlay();
         // set the map to be displayed in this view
         mMapView.setMap(mMap);
-
-        // work with the MapView after it has loaded
-        mMapView.addVisibleAreaChangedListener(new VisibleAreaChangedListener() {
-            @Override
-            public void visibleAreaChanged(VisibleAreaChangedEvent visibleAreaChangedEvent) {
-                //remove the listener so it's only called once
-                mMapView.removeVisibleAreaChangedListener(this);
-                // add graphics overlay
-                addGraphicsOverlay();
-            }
-        });
     }
 
     private void addGraphicsOverlay(){
-        // get center of MapView
-        Polygon visibleArea = mMapView.getVisibleArea();
-        Envelope polygonExtent = visibleArea.getExtent();
-        Point center = polygonExtent.getCenter();
-
-        // create values inside the visible area extent for creating graphic
-        double xValue = mMapView.getVisibleArea().getExtent().getWidth() / 5;
-        double yValue = mMapView.getVisibleArea().getExtent().getHeight() / 10;
-
-        // create point collection
-        PointCollection polyPoints = new PointCollection(mMapView.getSpatialReference());
-        polyPoints.add(new Point(center.getX() - xValue * 2, center.getY() - yValue * 2));
-        polyPoints.add(new Point(center.getX() - xValue * 2, center.getY() + yValue * 2));
-        polyPoints.add(new Point(center.getX() + xValue * 2, center.getY() + yValue * 2));
-        polyPoints.add(new Point(center.getX() + xValue * 2, center.getY() - yValue * 2));
-
-        // create graphics overlay
-        GraphicsOverlay grOverlay = new GraphicsOverlay();
-
-        // create list of graphics
-        ListenableList<Graphic> graphics = grOverlay.getGraphics();
-
-        // add points from PointCollection to graphics list
-        for(Point pt : polyPoints){
-            // add graphic to graphics overlay
-            graphics.add(new Graphic(pt));
-        }
-        // create color for graphic
-        int yellow = Color.rgb(255, 255, 0);
+        // point graphic
+        Point pointGeometry = new Point(40e5, 40e5, SpatialReferences.getWebMercator());
+        // yellow diamond point symbol
+        SimpleMarkerSymbol pointSymbol = new SimpleMarkerSymbol(Color.YELLOW, 10, SimpleMarkerSymbol.Style.DIAMOND);
```

Comment:
This color is quite difficult to see on top of the specified basemap - I thought to begin that there wasn't a Point showing up! For the polygon below it's clearer because the polygon takes up a noticeable amount of the map area. Change to Red or similar?